test/load/terraform/variables.tf

```diff
@@ -106,6 +106,5 @@ variable "consul_download_url" {
 variable "consul_version" {
   type        = string
   description = "Version of the Consul binary to install"
-  default     = "1.9.0"
+  default     = "1.12.0"
```

Comment:
I don't actually think this gets used for CI triggered jobs.

Once the EC2 instances are running they execute the user-data scripts which will download a version of consul. 

https://github.com/hashicorp/consul/blob/2427e3883914f14777eb0077c0230e403a54037f/test/load/terraform/user-data-server.sh#L14-L18

If a download URL is provided  then we ignore the consul_version and just pull whatever URL we were given. Our circle ci paramters cause this to be the case:

https://github.com/hashicorp/consul/blob/2427e3883914f14777eb0077c0230e403a54037f/.circleci/config.yml#L991

Reply:
Thanks for clarifying this!